phpcs.xml

```diff
@@ -30,7 +30,7 @@
 	<rule ref="PHPCompatibilityWP"/>
 	<config name="testVersion" value="5.6-"/>
 
-	<arg name="extensions" value="php,js"/>
```

Comment:
Tweak: I've removed the js check from phpcs.xml because phpcs freaks out on React code and thinks that e.g. `<div>` is using greater/less-than operators, so it generates hundreds of style errors. Additionally, now that we have prettier working, we probably don't need this. We can re-approach after this is merged if we want to add it back.